src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs

```diff
@@ -103,5 +98,16 @@ void Initialize()
 
 		TranslatesAutoresizingMaskIntoConstraints = false;
 		AddSubview(Container);
+
+		var subView = new RoundedView(
+			VisualOptions.CornerRadius.X,
+			VisualOptions.CornerRadius.Y,
+			VisualOptions.CornerRadius.Width,
+			VisualOptions.CornerRadius.Height)
+		{
+			BackgroundColor = VisualOptions.BackgroundColor,
+			AutoresizingMask = UIViewAutoresizing.FlexibleHeight | UIViewAutoresizing.FlexibleWidth
+		};
+		Subviews.First().InsertSubview(subView, atIndex: 0);
```

Comment:
Can we use Subviews[0]?

Reply:
Hi, @VladislavAntonyuk 
Thank you for your review.
It is possible to change to the proposed implementation.
I change and commit again.